app/src/androidTest/java/com/github/sdp/ratemyepfl/activity/CourseReviewActivityTest.kt

```diff
@@ -83,14 +74,15 @@ class CourseReviewListActivityTest {
         )
         val intent = Intent(
             ApplicationProvider.getApplicationContext(),
-            CourseReviewListActivity::class.java
+            CourseReviewActivity::class.java
         )
-        val scenario: ActivityScenario<CourseReviewListActivity> =
+        val scenario: ActivityScenario<CourseReviewActivity> =
             ActivityScenario.launch(intent)
+        onView(withId(R.id.courseReviewNavigationView)).perform(CustomViewActions.navigateTo(R.id.reviewListFragment))
+        Thread.sleep(500)

```

Comment:
I need to pause between the navigation and the change of the fakeList, otherwise the new list is directly displayed. I don't know exactly why that happens, but pausing fixes it.